TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/StrafeDrive.java

```diff
@@ -32,6 +40,27 @@ public void init() {
     // init_loop() - Runs continuously until the driver hits play
     @Override
     public void init_loop() {
+        if (gamepad1.a == true) {
+
+            if (gamepad1.dpad_down == true) {
+                tSpeed -= 1;
+            } else if (gamepad1.dpad_up == true) {
+                tSpeed += 1;
+            } else if (gamepad1.b == true) {
```

Comment:
dear lord fix your indentation. Dont you fkn say I'm acting like Milo, This is just horrendous lmao

Reply:
Gracious professionalism. /s

I agree though.